<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TPSystemeClocheClass</class>
 <widget class="QMainWindow" name="TPSystemeClocheClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TPSystemeCloche</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Déconnexion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cloche1Button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cloche 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cloche2Button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cloche 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cloche3Button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cloche 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cloche4Button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cloche 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="portEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="widgetHistorique">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>30</y>
      <width>291</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="clavierEdit">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>250</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Controle clavier</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Cloche 1 : 1
Cloche 2 : 2
Cloche 3 : 3
Cloche 4 : 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="songOn">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Douce
 Mélodie</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TPSystemeCloche.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>TPSystemeClocheClass</receiver>
   <slot>connexion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>disconnectButton</sender>
   <signal>clicked()</signal>
   <receiver>TPSystemeClocheClass</receiver>
   <slot>disconnexion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cloche3Button</sender>
   <signal>clicked()</signal>
   <receiver>TPSystemeClocheClass</receiver>
   <slot>cloche3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cloche2Button</sender>
   <signal>clicked()</signal>
   <receiver>TPSystemeClocheClass</receiver>
   <slot>cloche2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cloche1Button</sender>
   <signal>clicked()</signal>
   <receiver>TPSystemeClocheClass</receiver>
   <slot>cloche1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cloche4Button</sender>
   <signal>clicked()</signal>
   <receiver>TPSystemeClocheClass</receiver>
   <slot>cloche4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clavierEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TPSystemeClocheClass</receiver>
   <slot>clavierChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>songOn</sender>
   <signal>clicked()</signal>
   <receiver>TPSystemeClocheClass</receiver>
   <slot>songOn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connexion()</slot>
  <slot>disconnexion()</slot>
  <slot>cloche1()</slot>
  <slot>cloche2()</slot>
  <slot>cloche3()</slot>
  <slot>cloche4()</slot>
  <slot>clavierChange()</slot>
  <slot>songOn()</slot>
  <slot>songOff()</slot>
 </slots>
</ui>
